common/utils/works.js

```diff
@@ -234,6 +234,14 @@ export function getItemIdentifiersWith(
   }, []);
 }
 
+export type ArchiveNode = {|
+  id: string,
+  title: string,
+  alternativeTitles: string[],
+  referenceNumber: string,
+  type: 'Work',
+|};
```

Comment:
Could we start porting this to TS?

Reply:
Will do, but in another PR. This is still a prototype, so will make the changes if/when we pursue it, after testing.